Tests/PSSwaggerScenario.Tests.ps1

```diff
@@ -807,7 +812,7 @@ Describe "Header scenario tests" -Tag @('Header','ScenarioTest')  {
         $GeneratedModuleVersionPath = Join-Path -Path $GeneratedModuleBase -ChildPath $ModuleVersion
         $Header = '__Custom_HEADER_Content__'
         if((Get-Variable -Name PSEdition -ErrorAction Ignore) -and ('Core' -eq $PSEdition)) {
-            & "powershell.exe" -command "& {`$env:PSModulePath=`$env:PSModulePath_Backup;
+            & "$global:fullPowerShellPath\powershell.exe" -command "& {`$env:Path = '$global:fullPowerShellPath;$env:Path';`$env:PSModulePath=`$env:PSModulePath_Backup;
```

Comment:
>$env:Path = '$global:fullPowerShellPath;$env:Path' [](start = 72, length = 50)

May I know the reason for this?

Reply:
With recent builds of PSCore, running "powershell" in PSCore results in more PSCore.